openwisp_radius/tests/test_api.py

```diff
@@ -177,7 +238,11 @@ def test_postauth_400(self):
     def test_postauth_no_token_403(self):
         response = self.client.post(reverse('radius:postauth'), {})
         self.assertEqual(response.status_code, 403)
-        self.assertEqual(response.data, {'detail': 'Token authentication failed'})
+        self.assertEqual(
+            response.data['detail'],
+            'Radius token does not exist. Obtain new token or provide '
+            'API token and organization UUID.',
```

Comment:
What about: "Obtain a new radius token or provide the organization UUID and API token."?

Reply:
Done! :smile: